src/FTPC.jl

```diff
@@ -62,19 +61,29 @@ end
 
 function RequestOptions(
     url::AbstractString;
-    ssl::Bool=false,
     verify_peer::Bool=true,
     active_mode::Bool=false,
+    ssl::Union{Nothing, Bool}=nothing,
 )
+    if ssl !== nothing
+         Base.depwarn(
+            "RequestOptions(url; ssl=false, verify_peer=true, active_mode=false) is " *
```

Comment:
Code in deprecations usually has backticks around it. I think this can be made clearer with:
```julia
Base.depwarn(
	"`RequestOptions` keyword `ssl` has been deprecated, change the URL protocol " *
	"to \"ftp://\", \"ftps://\", or \"ftpes://\" to respectively indicate no " *
	"security, implicit security, or explicit security.",
	:RequestOptions,
)
```